app/models/user.rb

```diff
@@ -86,7 +86,7 @@ def self.add_friends(token, user_id)
         # {"name"=>"Sunjay Dhama", "id"=>"10205306719984646"}
         @user_friends = User.where(["uid = '%s'", hash["id"].to_s])
         @user_friends.each do |friend|
-          Friendship.create(friend_id: friend.id, user_id: user_id)
+          Friendship.create(friend_id: friend.id, user_id: self.user_id)
```

Comment:
@sdsunjay again, you don't need self unless you're writing over it, not reading. but pick what you like, its a preference to most ruby devs.